deployer-proposer.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:14.17
+
+WORKDIR /app
+COPY common-files common-files
+COPY config/default.js config/default.js
+COPY config/default.js /app/test/ping-pong/proposer/config/default.js
+COPY cli cli
+WORKDIR /app/common-files
+RUN npm ci
+WORKDIR /app/cli
+RUN npm ci
+
+WORKDIR /app/test/ping-pong/proposer
+RUN apt-get update -y
+RUN apt-get install -y netcat-openbsd
+COPY test/ping-pong/proposer/package*.json ./
+COPY test/ping-pong/proposer/src src
+COPY genesis-blocks/docker-entrypoint.sh docker-entrypoint.sh
```

Comment:
Similar to proposer in ping-pong where proposer is only waiting for optimist to start, this will also do the same. 
It does not need to wait for client and deployer to start because proposer will work independent of client.

In fact, we can reuse `proposer.Dockerfile`. This is not needed.